pilot/pkg/model/context.go

```diff
@@ -624,9 +624,9 @@ type NodeMetadata struct {
 	// This depends on DNSCapture.
 	DNSAutoAllocate StringBool `json:"DNS_AUTO_ALLOCATE,omitempty"`
 
-	// EnableHBONE, if set, will enable generation of HBONE config.
+	// ListenHBONE, if set, the proxy will listen on HBONE port.
 	// Note: this only impacts sidecars; ztunnel and waypoint proxy unconditionally use HBONE.
-	EnableHBONE StringBool `json:"ENABLE_HBONE,omitempty"`
+	ListenHBONE StringBool `json:"LISTEN_HBONE,omitempty"`
```

Comment:
This change may break backward compatability when doing inplace upgrade

Reply:
In place upgrade is not supported by istio, and this is also an experimental flag.